unison-share-api/src/Unison/Server/Backend.hs

```diff
@@ -8,10 +8,88 @@
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE ViewPatterns #-}
 
-module Unison.Server.Backend where
+module Unison.Server.Backend
+  ( -- * Types
+    BackendError (..),
+    Backend (..),
+    ShallowListEntry (..),
+    BackendEnv (..),
+    TypeEntry (..),
+    FoundRef (..),
+    NameScoping (..),
+    IncludeCycles (..),
+    DefinitionResults (..),
+
+    -- * Endpoints
+    prettyDefinitionsForHQName,
+    fuzzyFind,
+
+    -- * Utilities
+    basicSuffixifiedNames,
+    bestNameForTerm,
+    bestNameForType,
+    definitionsBySuffixes,
+    displayType,
+    docsInBranchToHtmlFiles,
+    expandShortCausalHash,
+    findShallowReadmeInBranchAndRender,
+    formatSuffixedType,
+    getCurrentParseNames,
+    getCurrentPrettyNames,
+    getShallowCausalAtPathFromRootHash,
+    getTermTag,
+    getTypeTag,
+    hoistBackend,
+    hqNameQuery,
+    loadReferentType,
+    loadSearchResults,
+    lsAtPath,
+    lsBranch,
+    mungeSyntaxText,
+    namesForBranch,
+    parseNamesForBranch,
+    prettyNamesForBranch,
+    resolveCausalHashV2,
+    resolveRootBranchHashV2,
+    scopedNamesForBranchHash,
+    termEntryDisplayName,
+    termEntryHQName,
+    termEntryToNamedTerm,
+    termEntryType,
+    termListEntry,
+    termReferentsByShortHash,
+    typeDeclHeader,
+    typeEntryDisplayName,
+    typeEntryHQName,
+    typeEntryToNamedType,
+    typeListEntry,
+    typeReferencesByShortHash,
+    typeToSyntaxHeader,
+    renderDocRefs,
+    docsForTermName,
+
+    -- * Unused, could remove?
```

Comment:
Discovered these aren't used anywhere ATM, just left them there for now, but they could be removed if we like.